gdk-sys/src/lib.rs

```diff
@@ -3369,16 +3553,31 @@ pub struct GdkEventScroll {
     pub delta_x: c_double,
     pub delta_y: c_double,
     _truncated_record_marker: c_void,
-    //is_stop: guint: 1,
+    // /*Ignored*/field is_stop has incomplete type
```

Comment:
This looks a bit weird an IMHO the previous version had some useful further information